src/components/Menu/MenuDivider/MenuDivider.tsx

```diff
@@ -1,9 +1,15 @@
-import PropTypes from "prop-types";
 import Divider from "../../Divider/Divider";
 import { backwardCompatibilityForProperties } from "../../../helpers/backwardCompatibilityForProperties";
+import { VibeComponentProps } from "../../../types";
+import { FC } from "react";
 import "./MenuDivider.scss";
 
-const MenuDivider = ({
+interface MenuDividerProps extends VibeComponentProps {
+  /** Backward compatibility for props naming **/
+  classname?: string;
+}
+
+const MenuDivider: FC<MenuDividerProps> & { isMenuChild?: boolean } = ({
```

Comment:
Why not include this in `MenuDividerProps`?

Reply:
Cause all of them are static properties. In other places we've done the same, so it's just for consistency